src/app/src/components/Timer.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { Box } from 'rebass';
+import { themeGet } from 'styled-system';
+
+import Text from './Text';
+import { saveSecondsToCompleteQuiz } from '../actions';
+
+const StyledTimer = styled(Box)`
+    position: absolute;
+    background-image: linear-gradient(
+        to right,
+        ${themeGet('colors.teals.0')} 30%,
+        ${themeGet('colors.teals.1')} 70%,
+        ${themeGet('colors.teals.2')} 100%
```

Comment:
This is cool

Reply:
right? though i swiped it from another component that @alexelash designed, so i can't take any credit